.github/workflows/docker.yml

```diff
@@ -67,9 +67,9 @@ jobs:
           targets: ${{ matrix.target.name }}
 
       - name: Setup node
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: 18
+          node-version: 'latest'
```

Comment:
@lucasfernog do you need a specific version here?

Reply:
I am just going to back out changes here... devs with a more a more informed opinion should drive change here.